index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+</head>
+<body>
+
+<!--Start navbar-sec-sec-->
+    <nav class="navbar-sec">
+        <div class="welcome">Welcome</div>
+        <div class="icon-menu"> <img src="img/menu-icon.svg" alt=""></div>
+    </nav>
```

Comment:
- [ ] This whole section should be inside a `header` element, kindly change that.
- [ ] The `nav` element should only have the menu button the logo should be outside of the `nav` element but still inside the `header`, please fix that. 😁
- [ ] The logo should be a link, so please wrap it with an `a` tag (don't forget to take care of the styling because wrapping it up with an `a` tag will change its style.

[OPTIONAL]

- [ ] I suggest that you put your name or your logo instead of `welcome`, because that's its purpose.

Reply:
I have done this